src/components/tests/content-tab-assistant.test.tsx

```diff
@@ -343,18 +389,29 @@ describe( 'ContentTabAssistant', () => {
 		);
 
 		const { rerender } = render( <ContentTabAssistant selectedSite={ runningSite } /> );
-		expect( screen.getByText( 'Welcome to our service!' ) ).toBeVisible();
-		expect( screen.getByText( 'Initial message 1' ) ).toBeVisible();
-		expect( screen.getByText( 'Initial message 2' ) ).toBeVisible();
-		expect(
-			screen.getByText( 'This conversation is over two hours old.', { exact: false } )
-		).toBeVisible();
+		act( () => {
+			jest.advanceTimersByTime( MIMIC_CONVERSATION_DELAY + 1000 );
+		} );
+		await waitFor(
+			() => {
+				expect( screen.getByText( 'Welcome to our service!' ) ).toBeVisible();
+				expect( screen.getByText( 'Initial message 1' ) ).toBeVisible();
+				expect( screen.getByText( 'Initial message 2' ) ).toBeVisible();
+				expect(
+					screen.getByText( 'This conversation is over two hours old.', { exact: false } )
+				).toBeVisible();
+			},
+			{ timeout: MIMIC_CONVERSATION_DELAY + 2000 }
+		);
 
 		( usePromptUsage as jest.Mock ).mockReturnValue( {
 			userCanSendMessage: false,
 			daysUntilReset: 4,
 		} );
 		rerender( <ContentTabAssistant selectedSite={ runningSite } /> );
+		act( () => {
+			jest.advanceTimersByTime( MIMIC_CONVERSATION_DELAY + 1000 );
+		} );
```

Comment:
I think we don't need to advance timers as in this test case we haven't submitted a prompt.

Reply:
fixed